### ISSUE TITLE
Fix `block-closing-brace-empty-line-before` with `after-closing-brace` false negatives for CSS Nesting

### ISSUE THREAD

**Issue by cdtut** [stylelint-stylistic/stylelint-stylistic#20]
### What minimal example or steps are needed to reproduce the bug?

```
.class {

  .class {
  }

}
```

### What minimal configuration is needed to reproduce the bug?

```
"@stylistic/block-closing-brace-empty-line-before": ["never", { "except": ["after-closing-brace"]}]
```

### How did you run Stylelint?

`stylelint "**/*.css"`

### Which Stylelint-related dependencies are you using?

@stylistic/stylelint-plugin 2.1.0
stylelint 16.3.1

### What did you expect to happen?

No error

### What actually happened?

Error on last line. If you change .class to an at rule then error is gone.

**Comment by firefoxic** [comment]
@cdtut Thanks for the report!

I found the cause and will fix it soon.

**Comment by firefoxic** [comment]
[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdllKBGaOKAvllqdpqhpegC4E332QA2EATvCkgI4CuSALZIOAS0wwA3GToVyWEABoQAMzGsYAOWFxEMAB7CADpoB0YAM6Xl4CJnUBzBCBq50IU-0cTLH7gDaRO4gAAKWdACerGIRYmAA9BHRMDGYdAC0Xj7YIEQAukpEHhz8mn4IhJi4IeFRMXGJeOxgANYZYOyWEo4ZeBxIYDAZMELGURlpw3gwqpww-ihB1TUhkgBuMBweRSurbqs1HoZD44vLh4ceSKp0Wx1dPX0DQx7Bh-nvuBSrnysUxFskAcYkcADFOCI6C4AFaWey2WDGGyIA4eZKaNJ0RYeVhIO4RHbFMIY2J0eJJeqpCSZbISHEgPEE7F5TCA4hAA) for testing.

**Comment by cdtut** [comment]
Now it has error for nested tags and nested selectors like `tag tag.class > tag::class`

```
.class {

  tag {
  }

  tag {
  }

}
```
